Sources/NIOPosix/SelectorKqueue.swift

```diff
@@ -381,7 +392,10 @@ extension kevent {
         self.filter = Int16(filter)
         self.flags = flags
         self.udata = UnsafeMutableRawPointer(bitPattern: UInt(registrationID.rawValue))
-
+        #if os(FreeBSD)
+        self.fflags = 0
+        self.data = 0
+        #else
```

Comment:
It's not clear to me that this is right: as noted above, EVFILT_EXCEPT does appear to be available on FreeBSD, and I think we actually do want it.